drawings.css

```diff
@@ -1,8 +1,10 @@
-#yourName {
```

Comment:
#yourName styles should not be overwritten, you can copy the code of #yourName and make changes in it.
Good luck